### DISCUSSION TITLE
A proper image and svg solution as alternative to FFImageloading

### DISCUSSION THREAD

**Discussion by EmilAlipiev** [xamarin/Xamarin.Forms#13039] [Ideas]
I appreciate your recent development and trying to make xamarin.forms better but it seems to me that basic requirements are neglected. It has been more than 5 years and xamarin.forms doesnt have a proper image library. we all have been using FFImageloading for years but obviously maintainers has no time to maintain it anylonger. since a year no single contribution on this library and problems are growing and growing. Now with XF 5, it became not usable. Is it not time for Xamarin to provide a better solution? What we need is 
- better image caching downloading
- better memory management
- displaying svg files
Basically what ffimageloading have done earlier at its best.

**Comment by holecekp** [comment]
SVG files: It would be nice if Xamarin.Forms would add support for SVG files for operation systems that supports this format natively. For example, UWP has SvgImageSource (https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.imaging.svgimagesource?view=winrt-19041). If the coresponding counterpart would be added into Xamarin.Forms, we could use SVG files for vector graphics on Windows. It would not add much code into Xamarin.Forms (probably) because OS would do the parsing and drawing.

On Android, this is not a big problem because SVG files can be converted to XML drawables supported by Android and they work well in Xamarin.Forms. Yes, this format is not so powerful but it suffices for most cases.

**Reply by hanls8** [reply]
very good

**Comment by Cfun1** [comment]
https://github.com/xamarin/Xamarin.Forms/issues/8078

**Comment by themronion** [comment]
you can use resizetizer nt

**Comment by GalaxiaGuy** [comment]
This discussion seems to have focussed on SVG support. There are other features of FFImageLoading I would like, for example loading placeholder, failure placeholder and fade on loaded.

**Comment by giuseppenovielli** [comment]
Move FFImageloading into Xamarin Community Toolkit.

**Reply by mcguu** [reply]
I do not know why I hate Xamarin Community Toolkit. may be it is prejudice.

**Comment by developer9969** [comment]
Hi is there anybody from the xamarin team monitoring this discussions? 
Absolutely we need support for SVG for starters..  lets get that in and then other features might follows. 
We totally rely on FFImageLoading and that project is kind of stale.. 

My cynical hat on I doubt anything will be done on this space.

**Comment by giuseppenovielli** [comment]
https://github.com/luberda-molinet/FFImageLoading/issues/1522#issue-774886088
